### PR TITLE
Homework 1 - scenario GivenWhenThen - Catch2

### PR DESCRIPTION
**Pull request by Marcinati**
Made by:
@lisie-sprawy
@Marcinati
@Tomonike
@WojtekMs

### CODE REVIEW

File: bowling/tests.ut.cpp [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    GIVEN("player make second move") {
+        WHEN("player knocks down all pins") {
+            THEN("pins is 0") {
+            }
+        }
+    }   
+}
+
+SCENARIO("if knocking down 10 pins", "[frames]") {
+    GIVEN("player make first move") {
```
Comment (WBigaj) [inline bowling/tests.ut.cpp:103]: Shouldn't this part of test be for second move and forward considering that knocking down all pins in first move is strike?

**Review by WBigaj — COMMENTED** [review]
I have checked your code and have one question.

File: bowling/tests.ut.cpp (hunk 1 above)
Comment (WojtekMs) [reply bowling/tests.ut.cpp:103]: I think that this test was about checking if pins count is 0 after knocking down 10 pins.

File: bowling/tests.ut.cpp [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    }
+    GIVEN("player makes second move") {
+        WHEN("player knocks down 10 pins in total") {
+            THEN("return spare") {
+            }
+        }
+    }
+    GIVEN("start frame") {
+        WHEN("player start new frame") {
+            THEN("player has 2 throw") {
```
Comment (ziobron) [inline bowling/tests.ut.cpp:75]: already covered by 4th test case

File: bowling/tests.ut.cpp (hunk 1 above)
Comment (ziobron) [reply bowling/tests.ut.cpp:103]: It can be merged with above. We will either count a knocked or staying pins

File: bowling/tests.ut.cpp [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+SCENARIO("if knocking down 10 pins", "[frames]") {
+    GIVEN("player make first move") {
+        WHEN("player knocks down all pins") {
+            THEN("all 10 pins down") {
+            }
+        }
+    }
+    GIVEN("player make first move") {
+        WHEN("player knocks all pins in first move") {
+            THEN("player has a strike") {
```
Comment (ziobron) [inline bowling/tests.ut.cpp:111]: looks like already covered in 3rd test case

**Review by ziobron — APPROVED** [review]
19 cases for me

**Comment by ziobron** [comment]
{
"requirement_ids": [3881, 1636]
"users": ["lisie-sprawy", "Marcinati", "Tomonike", "WojtekMs"] 
}

**Comment by ziobron** [comment]
{
"requirement_ids": [3881, 1636],
"users": ["lisie-sprawy", "Marcinati", "Tomonike", "WojtekMs"]
}

**Pull request closed without merging**